python/metatensor-operations/metatensor/operations/_dispatch.py

```diff
@@ -756,3 +875,95 @@ def zeros_like(array, shape: Optional[List[int]] = None, requires_grad: bool = F
         return np.zeros_like(array, shape=shape, subok=False)
     else:
         raise TypeError(UNKNOWN_ARRAY_TYPE)
+
+
+@torch_jit_script
+def sort_list_2(
+    data: List[Tuple[int, int]], reverse: bool = False
+) -> List[Tuple[int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_3(
+    data: List[Tuple[int, int, int]], reverse: bool = False
+) -> List[Tuple[int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_4(
+    data: List[Tuple[int, int, int, int]], reverse: bool = False
+) -> List[Tuple[int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_5(
+    data: List[Tuple[int, int, int, int, int]], reverse: bool = False
+) -> List[Tuple[int, int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_6(
+    data: List[Tuple[int, int, int, int, int, int]], reverse: bool = False
+) -> List[Tuple[int, int, int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_7(
+    data: List[Tuple[int, int, int, int, int, int, int]], reverse: bool = False
+) -> List[Tuple[int, int, int, int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_8(
+    data: List[Tuple[int, int, int, int, int, int, int, int]], reverse: bool = False
+) -> List[Tuple[int, int, int, int, int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_9(
+    data: List[Tuple[int, int, int, int, int, int, int, int, int]],
+    reverse: bool = False,
+) -> List[Tuple[int, int, int, int, int, int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
+    else:
+        return list(sorted(data))
+
+
+@torch_jit_script
+def sort_list_10(
+    data: List[Tuple[int, int, int, int, int, int, int, int, int, int]],
+    reverse: bool = False,
+) -> List[Tuple[int, int, int, int, int, int, int, int, int, int]]:
+    if reverse:
+        return list(sorted(data))[::-1]
```

Comment:
Python's sorted has a `reverse` flag as well: https://docs.python.org/3/library/functions.html#sorted

Reply:
not working with torch script for some reason

Reply:
crap. Ok, this is worth a comment as well then